docs/src/main/asciidoc/security.adoc

```diff
@@ -3,161 +3,185 @@ This guide is maintained in the main Quarkus repository
 and pull requests should be submitted there:
 https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 ////
-= Security Architecture and Guides
+= Quarkus Security overview
 
 include::./attributes.adoc[]
 
-Quarkus Security provides the architecture, multiple authentication and authorization mechanisms, and other tools for the developers to build a production-quality security for their Quarkus applications.
+Quarkus Security is a framework that provides the architecture, multiple authentication and authorization mechanisms, and other tools for you to build secure and production-quality Java applications.
 
-This document provides a brief overview of Quarkus Security and links to the individual guides.
+== Security architecture
 
-== Getting Started
```

Comment:
@michelle-purcell I thought of having something at the very top of the doc for the users quickly find a guide leading them to this guide where they can quickly create a secure Quarkus app, without even having to go through this whole doc. It would nice to have it retained somehow, more comments below re this point

Reply:
:+1: I moved 'Getting started down a little bit. I thought it made sense to come after the Architcture section, which is quite small. If the architecture information grows and we add diagrams etc then we revisit this. Thanks.